components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -81,6 +81,48 @@ To access these components for an organization, select **Chef Servers > Organiza
 
 A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuration and policy distribution. A cookbook contains recipes and other files, components, or directories.
 
+The Chef Infra Server UI lets you:
+
+- List out cookbooks with the latest version
+- View different versions of a cookbook
+- View the contents of a cookbook
+- View the details of a cookbook
+
+#### List of Cookbooks
+
+In Chef Infra Server, you can view all the cookbooks of an organization. Each Cookbook has its latest version mentioned in the list.
+
+{{< figure src="/images/automate/list-of-cookbooks.png" alt="List of Cookbooks">}}
+
+Select any cookbook from the list to view the details of that particular cookbook.
+
+#### View Different Versions of Cookbooks
+
+To view the different versions of the cookbook:
+
+- Select a specific cookbook from the list.
+- The interface contains a dropdown list for versions. Select any version to view the details.
```

Comment:
Another opportunity for parallel construction

Reply:
Provide a description that describes what the user needs to do instead of the screen. `Interface` is not an improvement over `UI`.

Study the Microsoft guide for [Describing Interactions with UI](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui) and revise.